### DISCUSSION TITLE
Неизвестный краш.

### DISCUSSION THREAD

**Discussion by md-exe** [snakers4/silero-models#235] [Q&A] [answered]
Привет. Где-то месяц назад экспериментировал с TTS, интегрировав его в GUI с возможностью выбора голоса, ввода текста и тд. Изначально, строя программу на PySimpleGUI, всё работало, как часы, но затем появилась проблема: если запустить программу и записать какую-то озвучку, повторное сохранение файла просто кладёт программу. Перезапуск всё решает, но эта "одноразовость" напрягает. По какой причине это может случаться? Дело точно не в перезаписи файлов, поскольку первый запуск программы успешно перезаписывает.

**Comment by md-exe** [comment]
Небольшая поправка - изначально визуал строился на Tkinter, но ни на нём, ни на PySimpleGUI программа всё ещё требует перезапуск для новой записи.

**Reply by snakers4** [reply, marked as answer]
https://github.com/snakers4/silero-models/issues/183

**Reply by md-exe** [reply]
Благодарю! Добавление к коду: torch._C._jit_set_profiling_mode(False) не только помогло решить проблему, но и ускорило процесс сохранения файла. :)
